orchestra/static/orchestra/todos/todo-checklist.html

```diff
@@ -22,4 +29,17 @@
       </span>
     </label>
   </div>
+  <ol ui-tree-nodes="" ng-model="todo.items" ng-class="{hidden: collapsed}">
+    <li ng-repeat="todo in todo.items | filter: {skipped_datetime: showSkipped?'!!':'!'}" ui-tree-node data-collapsed="true" ng-include="'nodes_renderer.html'"></li>
```

Comment:
What does `showSkipped?'!!':'!'` mean?

Reply:
`skipped_datetime: '!'` checks if the skipped_datetime is null

`skipped_datetime: '!!'` checks if the skipped_datetime is not null

`showSkipped?'!!':'!'` is choosing what to filter based on `showSkipped`

Reply:
if there's a less cryptic way to do this, i'm all for it :)